example/index.ts

```diff
@@ -50,7 +50,11 @@ document.addEventListener('DOMContentLoaded', async () => {
     newActiveLink.setAttribute('aria-current', 'page');
 
     // update displayed doc
-    exampleContainer.innerHTML = page.html;
+    exampleContainer.innerHTML = page.html.replace(
+      /\${package_version}/g,
+      // @ts-expect-error global variables
+      process.env.npm_package_version,
```

Comment:
This is working for me locally and it looks like it will work fine in the compiled version (I can see "0.5.4" in the compiled files). 🤞

Reply:
This is great, I didn't realise we have access to the version at build time! (not that it would be hard to extract, but it's good to have this "built in").